core/primitives/src/version.rs

```diff
@@ -235,7 +234,6 @@ impl ProtocolFeature {
             ProtocolFeature::FixStakingThreshold => 126,
             #[cfg(feature = "protocol_feature_fix_contract_loading_cost")]
             ProtocolFeature::FixContractLoadingCost => 129,
-            #[cfg(feature = "protocol_feature_ed25519_verify")]
             ProtocolFeature::Ed25519Verify => 131,
```

Comment:
Ah, this will need a lower protocol number now so that it is included in the stable version. Please move it up to the stable features and update the number accordingly.

Note that we have already bumped the protocol version to 58 in #8360, so you can just set the version to 58 and don't have to bump again.